crates/astria-conductor/src/types.rs

```diff
@@ -41,4 +75,152 @@ impl SequencerBlockSubset {
             rollup_transactions: our_rollup_data.transactions,
         })
     }
+
+    /// Return the block hash.
+    pub fn block_hash(&self) -> Hash {
+        self.block_hash
+    }
+
+    /// Return the header of the block.
+    pub fn header(&self) -> Header {
+        self.header.clone()
+    }
+
+    /// Get the height of the block.
+    #[must_use]
+    pub fn height(&self) -> Height {
+        self.header().height
+    }
+
+    /// Get the height of the block's parent.
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if the block height is less than or equal to 1.
+    /// Only the genesis block has a height of 1, and all other blocks must have
+    /// a larger height.
+    #[must_use]
+    pub fn parent_height(&self) -> Height {
+        assert!(
+            self.height().value() > 1,
+            "block height must be greater than 1"
+        );
+        Height::try_from(self.header().height.value() - 1)
+            .expect("should have been able to decriment tendermint height")
+    }
+
+    /// Get the height of the block's child, or the next block.
```

Comment:
```suggestion
    /// Get the next height relative to this block.
```
I thought this was a little confusing at first. "or the next block" intends the case when a block doesn't have a child, is that right?